qa/acceptance-tests/src/test/java/io/camunda/it/migration/IdentityDefaultEntitiesIT.java

```diff
@@ -68,9 +91,49 @@ void canMigrateOldDefaultEntities() {
             "camunda.migration.identity.management-identity.client-secret", IDENTITY_CLIENT_SECRET)
         .withProperty(
             "camunda.migration.identity.management-identity.audience",
-            CAMUNDA_IDENTITY_RESOURCE_SERVER)
-        .start();
+            CAMUNDA_IDENTITY_RESOURCE_SERVER);
+    camunda.start();
```

Comment:
💭  could we await it's actual termination? 🤔 not relevant here though, I can follow-up in the context of https://github.com/camunda/camunda/issues/33068